lib/index.js

```diff
@@ -57,6 +58,40 @@ function setupTracker(spec) {
   };
 }
 
+function getDataFiles(ecobalyseDataDir, checkDataDirIsSet = false) {
+  if (!ecobalyseDataDir && checkDataDirIsSet) {
+    throw new Error(`You need to specify ECOBALYSE_DATA_DIR env variable.
+It must point to a directory with the detailed versions of the processes files. Please, edit your .env file accordingly.`);
+  }
```

Comment:
Nit: "Well, actually" 😅 I think this shouldn't be part of that function with the added bloat, but rather checked in context before calling `checkDataFiles`.

I understand this is reaching nitpicking territory so I'll leave it up to you. Code works and that's probably all what matters 👍